addons/purchase/models/stock.py

```diff
@@ -179,3 +179,19 @@ def _quantity_in_progress(self):
         for poline in self.env['purchase.order.line'].search([('state','in',('draft','sent','to approve')),('orderpoint_id','in',self.ids)]):
             res[poline.orderpoint_id.id] += poline.product_uom._compute_quantity(poline.product_qty, poline.orderpoint_id.product_uom, round=False)
         return res
+
+    def action_view_purchase(self):
+        """ This function returns an action that display existing
+        purchase orders of given orderpoint.
+        """
+        action = self.env.ref('purchase.purchase_rfq')
+        result = action.read()[0]
+
+        # Remvove the context since the action basically display RFQ and not PO.
```

Comment:
Typo: Remove

Reply:
there is a filter in the original action that only display the po, not the rfq. on this stat button, we want to see everything, right?

Reply:
Agree with @sle-odoo, PO state should not matter here.